cmd/main.go

```diff
@@ -543,9 +546,9 @@ func setupDB(log logrus.FieldLogger) *gorm.DB {
 			log.WithError(err).Info("Failed to connect to DB, retrying")
 			return
 		}
-		db.DB().SetMaxIdleConns(0)
-		db.DB().SetMaxOpenConns(0)
-		db.DB().SetConnMaxLifetime(0)
+		db.DB().SetMaxIdleConns(Options.MaxIdleConns)
+		db.DB().SetMaxOpenConns(Options.MaxOpenConns)
+		db.DB().SetConnMaxLifetime(Options.ConnMaxLifetime)
```

Comment:
@filanov I remember something about having issues with the connections when the DB is restarted.
We had some issues around this in strato. Do you remember something about it?

Reply:
@ronniel1 it doesn't really change behavior. By default there are 2 idle connections up. In this case we will have 2 idle all the time and "Options.MaxIdleConns - 2" for half an hour by default. It will allow us not to spawn new connections in case of load and close them when load is gone.